### PR TITLE
Add details about CDN  plugin based assets

### PR DESCRIPTION
**Pull request by steinkel**

### CODE REVIEW

File: en/development/configuration.rst [hunk 1]
```diff
@@ -122,6 +122,23 @@ Asset.cacheTime
     <http://php.net/manual/en/function.strtotime.php>`_ can take.
     The default is ``+1 day``.
 
+Using a CDN
+------------------------------------
```
Comment (dereuromark) [inline en/development/configuration.rst]: too long header line?

File: en/development/configuration.rst [hunk 2]
```diff
@@ -122,6 +122,23 @@ Asset.cacheTime
...
+
+To use a CDN for loading your static assets, change ``App.imageBaseUrl``, ``App.cssBaseUrl``, 
+``App.jsBaseUrl`` to point the CDN URI, for example: ``https://mycdn.example.com/`` 
+(note the trailing ``/``).
+
+All images, scripts and styles loaded via HtmlHelper will prepend the absolute CDN path, matching 
+the same relative path used in the application. Please note there is a specific use case when using
+plugin based assets: plugins will not use the plugin's prefix when absolute BaseUrl URI is used, for example
```
Comment (dereuromark) [inline en/development/configuration.rst]: BaseUrl => backticked `jsBaseUrl` maybe?

File: en/development/configuration.rst (hunk 1 above)
Comment (markstory) [reply en/development/configuration.rst]: Too long is ok, too short causes build failures.

File: en/development/configuration.rst [hunk 3]
```diff
@@ -122,6 +122,23 @@ Asset.cacheTime
...
+``App.jsBaseUrl`` to point the CDN URI, for example: ``https://mycdn.example.com/`` 
+(note the trailing ``/``).
+
+All images, scripts and styles loaded via HtmlHelper will prepend the absolute CDN path, matching 
+the same relative path used in the application. Please note there is a specific use case when using
+plugin based assets: plugins will not use the plugin's prefix when absolute BaseUrl URI is used, for example 
+By default:
+
+    - ``$this->Helper->assetUrl('TestPlugin.logo.png')`` resolves to ``test_plugin/logo.png`` 
+If you set ``App.imageBaseUrl`` to ``https://mycdn.example.com/``:
```
Comment (markstory) [inline en/development/configuration.rst]:
```suggestion

If you set ``App.imageBaseUrl`` to ``https://mycdn.example.com/``: 
```

You need a newline for lists to render properly.

File: en/development/configuration.rst [hunk 4]
```diff
@@ -122,6 +122,24 @@ Asset.cacheTime
...
+(note the trailing ``/``).
+
+All images, scripts and styles loaded via HtmlHelper will prepend the absolute CDN path, matching 
+the same relative path used in the application. Please note there is a specific use case when using
+plugin based assets: plugins will not use the plugin's prefix when absolute ``...BaseUrl`` URI is used, for example 
+By default:
+
+    - ``$this->Helper->assetUrl('TestPlugin.logo.png')`` resolves to ``test_plugin/logo.png`` 
+
+es "cakedc/users": "dev-develop"If you set ``App.imageBaseUrl`` to ``https://mycdn.example.com/``:
```
Comment (markstory) [inline en/development/configuration.rst]: The beginning of this line looks like a mis-paste.

**Comment by steinkel** [comment]
fixed mis-paste, lucky it was not my PK ;)

**Pull request merged**